src/OpenTelemetry.Instrumentation.AWSLambda/AWSLambdaWrapper.cs

```diff
@@ -181,8 +186,8 @@ internal static Activity OnFunctionStart<TInput>(TInput input, ILambdaContext co
             }
         }
 
-        var functionTags = AWSLambdaUtils.GetFunctionTags(input, context);
-        var httpTags = AWSLambdaHttpUtils.GetHttpTags(input);
+        var functionTags = (IEnumerable<KeyValuePair<string, object?>>)AWSLambdaUtils.GetFunctionTags(input, context);
```

Comment:
Why do we need this cast now?

Reply:
I added null-forgiving to the tags argument of the StartActivity method. No more casting needed. It's removed.

Reply:
Can you please double-check if we actually have a potential null value here? Maybe some return value type needs to have `?` removed.

Reply:
If you mean methods `GetFunctionTags` and `GetHttpTags`. I checked them and they do null-check of values before adding.